google/cloud/security/scanner/samples/group_rules.yaml

```diff
@@ -0,0 +1,50 @@
+rules:
+  # Only allow Organization users to be in the Organization groups.
+  - name: sample whitelist for group users
+    mode: whitelist
+    resource:
+      - type: group
+        applies_to: self_and_children
+        resource_ids:
+          - my_customer
```

Comment:
Hmmm, I'm not sure how this works yet, downstream when we start to gather the gcp group data.  I assumed this was the initial entry point, to start gathering all the group.  So to that effect, I guess it is the "root" where everything rolls up to?